app/uk/gov/hmrc/agentclientrelationshipsfrontend/models/forms/createInvitation/ConfirmationForm.scala

```diff
@@ -21,12 +21,11 @@ import play.api.data.Forms.*
 import uk.gov.hmrc.agentclientrelationshipsfrontend.models.forms.helpers.FormFieldHelper
```

Comment:
can we move this form into the models/forms/journey folder? it is in a folder that was created before we started sharing things between journeys